docs/tutorial/cspc/tuning/tune.md

```diff
@@ -0,0 +1,305 @@
+
+      poolConfig:
+        dataRaidGroupType: mirror
+
+    - nodeSelector:
+        kubernetes.io/hostname: worker-node-2
+
+      dataRaidGroups:
+      - cspiBlockDevices:
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f39
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f40
+
+      poolConfig:
+        dataRaidGroupType: mirror
+
+    - nodeSelector:
+        kubernetes.io/hostname: worker-node-3
+
+      dataRaidGroups:
+      - cspiBlockDevices:
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f42
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f43
+
+      poolConfig:
+        dataRaidGroupType: mirror
+        resources:
+          requests:
+            memory: 70Mi
+            cpu: 300m
+          limits:
+            memory: 130Mi
+            cpu: 600m
+
+        auxResources:
+          requests:
+            memory: 60Mi
+            cpu: 500m
+          limits:
+            memory: 120Mi
+            cpu: 500m
+
+```
+
+## Toleration
+
+Tolerations are applied in a similar manner like `resources` and `auxResources`.
+The following is a sample CSPC YAML that has tolerations specified. For `worker-node-1` and `worker-node-2` tolerations are applied form @spec.tolerations but for `worker-node-3` it is applied from @spec.pools[2].poolConfig.tolerations
+ 
+```yml
+apiVersion: cstor.openebs.io/v1
+kind: CStorPoolCluster
+metadata:
+  name: demo-pool-cluster
+  namespace: openebs
+spec:
+
+  tolerations:
+  - key: data-plane-node
+    operator: Equal
+    value: true
+    effect: NoSchedule
+
+  pools:
+    - nodeSelector:
+        kubernetes.io/hostname: worker-node-1
+
+      dataRaidGroups:
+      - cspiBlockDevices:
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f36
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f37
+
+      poolConfig:
+        dataRaidGroupType: mirror
+
+    - nodeSelector:
+        kubernetes.io/hostname: worker-node-2
+
+      dataRaidGroups:
+      - cspiBlockDevices:
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f39
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f40
+
+      poolConfig:
+        dataRaidGroupType: mirror
+
+    - nodeSelector:
+        kubernetes.io/hostname: worker-node-3
+
+      dataRaidGroups:
+      - cspiBlockDevices:
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f42
+          - blockDeviceName: blockdevice-ada8ef910929513c1ad650c08fbe3f43
+
+      poolConfig:
+        dataRaidGroupType: mirror
+        tolerations:
+        - key: data-plane-node
+          operator: Equal
+          value: true
+          effect: NoSchedule
+
+        - key: apac-zone
+          operator: Equal
+          value: true
+          effect: NoSchedule
+```
+
+## Priority Class
+Priority Class are also applied in a similar manner like `resources` and `auxResources`.
```

Comment:
Priority Class are applied in a similar manner like #resource-and-limits

Reply:
#resource-and-limits are not the actual configurations in the structure that is why the statement is like that. 
Can you please elaborate your thoughts here?

Reply:
I meant to add a link #resource-and-limits instead of explicitly saying `resources` and `auxResources`.

Reply:
But that can be understandable.

Reply:
Ignore this comment